src/switch_ivr_play_say.c

```diff
@@ -1600,10 +1600,18 @@ SWITCH_DECLARE(switch_status_t) switch_ivr_play_file(switch_core_session_t *sess
 			}
 
 			if ((f = switch_channel_test_flag(channel, CF_BREAK))) {
+				switch_log_printf(SWITCH_CHANNEL_SESSION_LOG(session), SWITCH_LOG_INFO, "CF_BREAK, f = %d\n", f);
```

Comment:
why would be want this to be logged with SWITCH_LOG_INFO ? DEBUG is more appropiate. maybe not even log at all.